DelvEdit/src/com/interrupt/dungeoneer/editor/ui/EditorUi.java

```diff
@@ -346,7 +346,10 @@ public void actionPerformed(ActionEvent e) {
 
         menuBar.addItem(
             new MenuItem("Level", smallSkin)
-                .addItem(new MenuItem("Test Level", smallSkin, actions.playAction).setAccelerator(new MenuAccelerator(Keys.P, false, false)))
+                        .addItem(new MenuItem("Test Level From Camera", smallSkin, actions.playFromCameraAction)
```

Comment:
The hotkey is 'P' for 'Play' Maybe change the verbiage to:
- "Play From Camera"
- "Play From Start"

Reply:
Good idea!

Reply:
Changed. Closing.